optuna/study.py

```diff
@@ -472,14 +473,34 @@ def create_study(
         direction:
             Direction of optimization. Set ``minimize`` for minimization and ``maximize`` for
             maximization. Note that ``maximize`` is currently unsupported.
+        exist_ok:
+            Flag for controlling the behavior when study names conflict.
```

Comment:
Personally, `to control` seems to be a more common expression than `for controlling`.
I think we can rephrase the sentence as follows:
`Flag to control the behavior to handle a conflict of study names.`